Got too many pings from the client

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

1. Start a TiDB cluster.
2. Enable TopSQL.

<!-- a step by step guide for reproducing the bug. -->

### 2. What did you expect to see? (Required)

No warning/error.

### 3. What did you see instead (Required)

```
[ERROR] [grpclogger.go:116] ["[transport]transport: Got too many pings from the client, closing the connection."] [system=grpc] [grpc_log=true]"
[ERROR] [grpclogger.go:116] ["[transport]transport: loopyWriter.run returning. Err: transport: Connection closing"] [system=grpc] [grpc_log=true]
```

### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

ea26284e

--- a/server/rpc_server.go
+++ b/server/rpc_server.go
@@ -55,6 +55,12 @@ func NewRPCServer(config *config.Config, dom *domain.Domain, sm util.SessionMana
 			Time:    time.Duration(config.Status.GRPCKeepAliveTime) * time.Second,
 			Timeout: time.Duration(config.Status.GRPCKeepAliveTimeout) * time.Second,
 		}),
+		grpc.KeepaliveEnforcementPolicy(keepalive.EnforcementPolicy{
+			// Allow clients send consecutive pings in every 5 seconds.
+			// The default value of MinTime is 5 minutes,
+			// which is too long compared with 10 seconds of TiDB's keepalive time.
+			MinTime: 5 * time.Second,
+		}),
 		grpc.MaxConcurrentStreams(uint32(config.Status.GRPCConcurrentStreams)),
 		grpc.InitialWindowSize(int32(config.Status.GRPCInitialWindowSize)),
 		grpc.MaxSendMsgSize(config.Status.GRPCMaxSendMsgSize),

